deckhouse-controller/pkg/controller/loader.go

```diff
@@ -117,6 +118,11 @@ func (dml *DeckhouseController) processModuleDefinition(def models.DeckhouseModu
 		log.Debugf("conversions for %q module not found", valuesModuleName)
 	}
 
+	// Load constrains of installed modules
+	if err = extenders.AddInstalledConstraints(def.Name, def.Requirements); err != nil {
+		return nil, err
+	}
+
```

Comment:
It could be safer to add constraints only when we are sure that the definition in question is going to end up as a Deckhouse module, couldn't it? In the opposite case, there can be some strange cases when we have already added some constraints from a definition, but the method returns error instead of a Deckhouse module.

Reply:
seems logical, but in this case, do I need to move loading conversions as well?

Reply:
wow, yeah, I think it makes sense, for the sake of consistency